LEGO1/lego/legoomni/include/legotextureinfo.h

```diff
@@ -19,7 +19,7 @@ class LegoTextureInfo {
 	static BOOL SetGroupTexture(Tgl::Mesh* pMesh, LegoTextureInfo* p_textureInfo);
 	static BOOL GetGroupTexture(Tgl::Mesh* pMesh, LegoTextureInfo*& p_textureInfo);
 
-	LegoResult FUN_10066010(LegoU8* p_bits);
+	LegoResult FUN_10066010(const LegoU8* p_bits);
```

Comment:
I think this might be a change we can add to the decomp repo?

Reply:
Yup, I think so.